docs/extras/block-updates.md

```diff
@@ -1,6 +1,6 @@
 # Blocking Updates
 ---
-All currently known Wii U exploits can, unlike e.g. the Nintendo Switch RCM exploit, be patched by a system update. Although the Wii U is no longer officially supported, Nintendo may still release updates for it. Namely, the updates 5.5.3 and 5.5.4 were both released after the Wii U was discontinued, so blocking updates is still a recommended action.
+All currently known Wii U exploits can, unlike e.g. the Nintendo Switch RCM exploit, be patched by a system update. Although the Wii U is no longer officially supported, Nintendo may still release updates for it. Namely, the updates 5.5.3, 5.5.4 and 5.5.5 were both released after the Wii U was discontinued, so blocking updates is still a recommended action.
```

Comment:
Don't use "both" when referring to 3 updates.

Reply:
Ah okay